docs/maintenance.md

```diff
@@ -0,0 +1,22 @@
+# OSPS Baseline Maintenance Process
+
+* Normal text fixes to the criteria will be accepted via pull request and reviewed by the baseline project maintainers.
+Allowed changes are corrections to spelling/typos, grammar corrections, or enhancements to the supplementary text supporting the criteria, including: Objective, Implementation, and Control Mappings.
+At least two project maintainers must review and approve these changes.
+* Substantive changes to Criteria, including changes to text that alters the originally stated meaning, new Criteria proposals, or removal of Criteria will be documented in GitHub PR(s) and reviewed regularly by the Baseline project maintainers for inclusion in the next release.
+These changes may reflect changes to global cybersecurity regulations and frameworks or changes in norms around application/project security practices.
+Any such substantive changes must be approved by a majority of the project's maintainers.
+* As appropriate, but at least annually, the Baseline project maintainers will evaluate the set of criteria and, if necessary, publish a new version of the Baseline.
+Previous versions of the Baseline will remain available, but are stable and not subject to change, except for minor changes to fix technical or typographic errors.
+* Any changes to the Baseline will be reflected within the Compliance Matrix, with new requirements flagged where the Baseline Criteria are appropriate.
+* Versions will follow a calendar-based identification system, using the `YYYY-MM-DD` format.
+* Downstream stakeholders will be notified via the project's mailing list on the changes and updates.
+
+## Identifiers
+
+* Identifiers for retired criteria MUST NOT be reused.
+Retired identifiers will remain in the source yaml files, clearly marked.
+* Substantial changes to the meaning of a criterion will be treated as a new criterion, resulting in a new identifier.
+Minor changes, including a change in level, between Baseline versions will not result in a new identifier.
+* The numeric portion of identifiers are assigned sequentially per category.
+They do not carry additional meaning.
```

Comment:
@SecurityCRob can you suggest a rephrase here to make it fit with your proposed three digit change? (ie, the levels DO carry additional meaning)

Reply:
Are we fully set on the three digit change? The notes from last week say "try doing this with one category, and look at it." (although Crob has done all of the categories) and I, at least, am still not sold on the idea.

In the interests of getting this actually done, let's assume for now that the numbers aren't changing and we can update this if/when they do.

Reply:
How about this:

```suggestion
* The numeric portion of identifiers are assigned sequentially per category and level.
Within a level (i.e. 201, 202, etc), numbers do not carry a meaning, but should not be re-used across versions.
Moving a criteria between levels will (intentionally) result in assigning a new number, to allow tracking changes to levels over time.
```